addons/web/controllers/webclient.py

```diff
@@ -87,7 +87,11 @@ def translations(self, hash=None, mods=None, lang=None):
 
     @http.route('/web/webclient/version_info', type='jsonrpc', auth="none")
```

Comment:
Should we deprecate this one? use /json/version instead

Reply:
It doesn't hurt to have one `type='jsonrpc'` function that does it. I don't care too much personally.